ld/zld.xcodeproj/project.pbxproj

```diff
@@ -1240,14 +1206,15 @@
 			buildSettings = {
 				CLANG_CXX_LANGUAGE_STANDARD = "c++17";
 				CLANG_CXX_LIBRARY = "libc++";
+				CODE_SIGNING_ALLOWED = NO;
```

Comment:
why this?

Reply:
Otherwise it is impossible to build the linker without specifying a development team, which cannot be checked in to the repo. Still worked fine for me without codesigning, I'm not entirely clear on the rules for command line tools though. @kastiglione any ideas on this one, can we just skip this entirely? What is the portable way to handle codesigning Xcode projects with different development teams?